packages/mongodb/db/index.js

```diff
@@ -0,0 +1,43 @@
+export default async ({
+  config: {
+    mongodb: {
+      user,
+      password,
+      url,
+      databases,
+      poolSize = 20
+    },
+    events: { shutdown }
+
+  },
+  dependencies: {
+    logger,
+    events,
+    mongodb: {
+      MongoClient
+    }
+  }
+}) => {
+  try {
+    const client = await MongoClient.connect(url, {
+      auth: user ? {
+        password,
+        user
+      } : undefined,
+      poolSize,
+      useUnifiedTopology: true
+    })
+    process.env.UV_THREADPOOL_SIZE = Number(poolSize)
```

Comment:
this was a quick patch we did for a LM service.  Not sure if this should be handled a bit more elegantly

Reply:
We may not want to include that, placing a number that exceeds the node UV_THREADPOOL_SIZE could become problematic

Reply:
do you remember what this was for exactly? maybe we can just remove it all together.